data/data/fcos.json

```diff
@@ -1,166 +1,149 @@
-            "sha256": "ace6a2226cb9142cdcc41f00849bcbe45e761f9c9b339e7fa4a9ed79fadf363b",
-            "size": 734159676,
-            "uncompressed-sha256": "aa8e537f7fe5644ade9f58d67fddda705b9b99bbfe10f4f52d52906f0de34996",
-            "uncompressed-size": 775048192
+            "path": "fedora-coreos-32.20200601.3.0-aws.x86_64.vmdk.xz",
+            "sha256": "5683bad6307ea1626725c79fc88f38ba0ac32c847755b3b2f3e126ed47a66cd5",
+            "size": 734917852,
+            "uncompressed-sha256": "d209a6807936b306a1612261bfdd4f3653bd0a565a453cbc6296c267f9526f07",
+            "uncompressed-size": 775761408
         },
         "azure": {
-            "path": "fedora-coreos-32.20200517.1.0-azure.x86_64.vhd.xz",
-            "sha256": "fad529db18348874827d4ef8522ff36748dedca723355e4d60069045e39be3ab",
-            "size": 504146464,
-            "uncompressed-sha256": "6c5e1394fe62dee5af7dca9137b16226fb7aa09f6836cea66de025d9b9d932f8",
+            "path": "fedora-coreos-32.20200601.3.0-azure.x86_64.vhd.xz",
+            "sha256": "7d7abd74f32ee159e90bd23ebbbb882d9716cb176a0349af23876fc705820132",
+            "size": 503626396,
+            "uncompressed-sha256": "0990669f182eb1241716d0aa50207a38cf2551bd3ad6b154e5af57d1848e05d3",
             "uncompressed-size": 8589935104
         },
-        "digitalocean": {
-            "path": "fedora-coreos-32.20200517.1.0-digitalocean.x86_64.qcow2.gz",
-            "sha256": "4c10662859cc7cc6a43977fcc252afb164a2c1b5011f88052290dcebbee7686c",
-            "size": 755027969
-        },
-        "exoscale": {
-            "path": "fedora-coreos-32.20200517.1.0-exoscale.x86_64.qcow2.xz",
-            "sha256": "e848f301b01270df0f188fe8d07907897b1cf03fc4f899275489f7b20a651a22",
-            "size": 503465784,
-            "uncompressed-sha256": "908c334fda245c61b66dcbf4b34c61eef645fb1a5dd164baa47b92df886828ff",
-            "uncompressed-size": 1774845952
-        },
         "gcp": {
-            "path": "fedora-coreos-32.20200517.1.0-gcp.x86_64.tar.gz",
-            "sha256": "b64998376b63a9408ffcc3d91eee8e077628008de70e47160603818806521db1",
-            "size": 757806712
+            "path": "fedora-coreos-32.20200601.3.0-gcp.x86_64.tar.gz",
+            "sha256": "17605fc24e9531e319ee310684beb48cd2687a681e1082d130049eb3b7372748",
+            "size": 758473229
         },
-        "live-initramfs": {
-            "path": "fedora-coreos-32.20200517.1.0-live-initramfs.x86_64.img",
-            "sha256": "15f619bd80a7a1e788127b2052a848a432ffb181d562a1afa662259c75e85203"
+        "initramfs": {
+            "path": "fedora-coreos-32.20200601.3.0-live-initramfs.x86_64.img",
+            "sha256": "08621f5f8c07128ee9708055e765dd8cb9006538cc48bbb2f9410fcca7aba5d0"
         },
-        "live-iso": {
-            "path": "fedora-coreos-32.20200517.1.0-live.x86_64.iso",
-            "sha256": "785488d8fab61653fd1b140a91ce20a5661c6e0bc2483ce3d73a2c47b1ef736f"
+        "iso": {
```

Comment:
Huh that's strange, it changed from `live-iso` to `iso`.  That...might be a bug.

Reply:
Installer (from `release-4.5`) wants `iso`, although FCOS has `live-iso`, isn't it? Pushed a few commits to update `./hack/update-fcos-bootimage.py`